core/build.gradle

```diff
@@ -0,0 +1,31 @@
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 26
+
+    defaultConfig {
+        minSdkVersion 21
```

Comment:
[Idea] If we want to keep this consistent across all sdk modules we can use constants and import them.
For example here:
https://github.com/facebook/facebook-android-sdk/blob/master/constants.gradle